clients/ui/frontend/src/app/ThemeContext.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+type ThemeContextProps = {
+  isMUITheme: boolean;
+};
+
+export const ThemeContext = React.createContext({
+  isMUITheme: true,
```

Comment:
@jenny-s51 I think this should be set to false to match the default component library being used. And the default style.

Reply:
Thanks @christianvogt! Updated to `false`